examples/gameserverallocation.yaml

```diff
@@ -31,33 +31,32 @@ spec:
   # flexibility.
   # Below is an example of a GameServer allocated against a given fleet.
   # See: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/ for more details
-  required:
-    matchLabels:
-      game: my-game
-    matchExpressions:
-      - {key: tier, operator: In, values: [cache]}
-    # [Stage:Alpha]
-    # [FeatureFlag:StateAllocationFilter]
-    # Specifies which State is the filter to be used when attempting to retrieve a GameServer
-    # via Allocation. Defaults to "Ready". The only other option is "Allocated", which can be used in conjunction with
-    # label/annotation/player selectors to retrieve an already Allocated GameServer.
-    gameServerState: Ready
-    # [Stage:Alpha]
-    # [FeatureFlag:PlayerAllocationFilter]
-    # Provides a filter on minimum and maximum values for player capacity when retrieving a GameServer
-    # through Allocation. Defaults to no limits.
-    players:
-      minAvailable: 0
-      maxAvailable: 99
   # ordered list of preferred allocations out of the `required` set.
```

Comment:
nit: remove "out of the 'required' set" here.

Reply:
Done.